### PR TITLE
Bump pyodide version to 0.25.0

### PR DESCRIPTION
**Pull request by bartbroere**
Bumping the Pyodide version. Seems to work okay, except the `.isPyProxy()` method now shows a deprecation warning.

TODO:
- [x] Iron out deprecation

### CODE REVIEW

File: packages/starboard-python/package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
   "dependencies": {
     "@types/katex": "^0.11.1",
     "lit": "^2.0.2",
-    "pyodide": "^0.20.0"
+    "pyodide": "^0.23.1"
```
Comment (bartbroere) [inline packages/starboard-python/package.json:34]: Bump some more :-)

```suggestion
    "pyodide": "^0.23.2"
```

**Comment by bartbroere** [comment]
The monorepo approach makes it a bit hard to test this, because it really likes to use old versions from npm, instead of the version it just built.

**Comment by bartbroere** [comment]
> The monorepo approach makes it a bit hard to test this, because it really likes to use old versions from npm, instead of the version it just built.

I got it going by doing a `yarn add ../starboard-python` in the `packages/starboard-notebook` directory. This shows me that it works:

image

As the CI currently runs it will build each package individually but it will not test the combination of the newly built packages, since `starboard-notebook` gets built with the NPM release of `starboard-python` and `starboard-rich-editor`. I needed to add the artifact upload in the CI to come to that conclusion. What gives it away is that it still pulls the 0.20.0 release of Pyodide from jsdelivr.net.

This pull request is in working order though.

**Comment by gzuidhof** [comment]
Thank you 🙏

**Pull request merged**